util/libcrypto.num

```diff
@@ -4796,3 +4796,24 @@ EVP_MD_fetch                            4743	3_0_0	EXIST::FUNCTION:
 EVP_set_default_properties              4744	3_0_0	EXIST::FUNCTION:
 OSSL_PARAM_construct_end                4745	3_0_0	EXIST::FUNCTION:
 EC_GROUP_check_named_curve              4746	3_0_0	EXIST::FUNCTION:EC
+OSSL_CMP_PKISTATUS_it                   4747	3_0_0	EXIST:!EXPORT_VAR_AS_FUNCTION:VARIABLE:CMP
+OSSL_CMP_PKISTATUS_it                   4748	3_0_0	EXIST:EXPORT_VAR_AS_FUNCTION:FUNCTION:CMP
+d2i_OSSL_CMP_PKIHEADER                  4749	3_0_0	EXIST::FUNCTION:CMP
+i2d_OSSL_CMP_PKIHEADER                  4750	3_0_0	EXIST::FUNCTION:CMP
```

Comment:
These two should be added to the list at the beginning of the d2i_X509.pod file. Similarly for the other d2i/i2d functions below.

Reply:
Done (see also above).